Signal/src/call/CallService.swift

```diff
@@ -81,6 +81,18 @@ enum CallError: Error {
 // Should be roughly synced with Android client for consistency
 fileprivate let connectingTimeoutSeconds = 120
 
+// Example: OWSProdError("blah", #file, #function, #line)
+func OWSProdError(_ __eventName: String, file __file: String, function __function: String, line __line: Int32) {
+    let location = "\((__file as NSString).lastPathComponent):\(__function)"
+    OWSAnalytics
+        .logEvent(__eventName, severity: .error, parameters: nil, location: (location as NSString).utf8String!, line:__line)
+}
```

Comment:
So far, I've only instrumented a single .swift file.  If we instrument anywhere else, I'll pull out a new OWSAnalytics.swift file.  

Is there any problem (at link time?) with having objective-c and swift modules with the same name, e.g. Alice.m and Alice.swift?